jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/core/ls/ArgumentUtils.java

```diff
@@ -139,20 +147,34 @@ public static Map<String, Object> getObject(Map<String, Object> obj, String key)
     }
 
     /**
-     * Returns the child as a JsonObject if it exists and is an object, and null
-     * otherwise
-     *
-     * @param obj the object to get the child of
-     * @param key the key of the child
-     * @return the child as a JsonObject if it exists and is an object, and null
-     *         otherwise
-     */
-    public static JsonObject getObjectAsJson(Map<String, Object> obj, String key) {
-        Object child = obj.get(key);
-        if (child != null && child instanceof Map<?, ?>) {
-            Gson gson = new Gson();
-            return (JsonObject) gson.toJsonTree(child);
-        }
-        return null;
+	 * Returns the child as a JSON array if the data parameter contains an array of
+	 * strings; otherwise,
+	 * returns it as an object if present, or null if not.
+	 *
+	 *
+	 * @param data the object to get the child of
+	 * @param key  the key of the child
+	 * @return Returns the child as a JSON array if the data parameter contains an
+	 *         array of strings; otherwise,
+	 *         returns it as an object if present, or null if not.
+	 */
+    public static Object getValueFromDataParameter(Map<String, Object> data, String key) {
+
+    	Object child = data.get(key);
+    	if (child instanceof String) {
+    		// if the value in the 'data' is a string, we string the object.
```

Comment:
Unfortunately it appears you added a tab to every line rather than change to spaces. Could you configure your editor to use spaces instead of tabs? Alternatively could you use a different editor like vi that will make it more obvious when you are using tabs and give you control over whether tabs or spaces are used?

Reply:
@turkeylurkey Please check now, I have corrected the indentation issue.